lib/src/widget/nav_bar/widget/nav_bar.dart

```diff
@@ -85,25 +70,17 @@ class NavBarState extends State<NavBar> implements ConfigUpdater {
       cardBackground: linden.colors.background,
     );
 
-    final sized = SizedBox(
-      child: Center(child: card),
-      height: linden.dimen.navBarHeight,
-    );
+    final sized = _withFixedHeight(card);
 
-    if (!widget.aboveTheKeyboard) {
-      return _withPadding(sized);
-    }
-
-    final future = Future<double>.delayed(
-      widget.keyboardCheckDelay,
-      () => MediaQueryData.fromWindow(WidgetsBinding.instance!.window)
-          .viewInsets
-          .bottom,
-    );
+    return config.showAboveKeyboard
+        ? _withPadding(sized, MediaQuery.of(context).viewInsets.bottom)
```

Comment:
💅 Here we might add a comment specifying that `MediaQuery.of(context).viewInsets.bottom` represents the height of the keyboard when it's shown :)

Reply:
yes, thank you, added comment